sdcm/utils/nemesis_jobs_configs.py

```diff
@@ -12,15 +12,3 @@
     "PauseLdapNemesis": ["configurations/ldap-authorization.yaml"],
     "ToggleLdapConfiguration": ["configurations/ldap-authorization.yaml"],
 }
-
-NEMESIS_ADDITIONAL_PIPELINE_PARAMS = {
```

Comment:
@juliayakovlev please bring this back

Reply:
did it. But actually we do not need it for AWS (this parameter is used for AWS only AFAIU). 
And if it does not exists - empty dict will be created:
```
additional_params = config.NEMESIS_ADDITIONAL_PIPELINE_PARAMS.get(cls, {})
```